Author: Daniel Baumann <daniel@debian.org>
Description: Adding another shebang path for bash.

diff -Naurp file.orig/magic/Magdir/commands file/magic/Magdir/commands
--- file.orig/magic/Magdir/commands	2008-04-16 12:28:04.000000000 +0000
+++ file/magic/Magdir/commands	2009-02-18 11:58:44.000000000 +0000
@@ -56,6 +56,10 @@
 # bash shell magic, from Peter Tobias (tobias@server.et-inf.fho-emden.de)
 0	string/b	#!\ /bin/bash	Bourne-Again shell script text executable
 !:mime	text/x-shellscript
+0	string/b	#!\ /usr/bin/bash	Bourne-Again shell script text executable
+!:mime text/x-shellscript
+0	string/b	#!\ /usr/local/bash	Bourne-Again shell script text executable
+!:mime text/x-shellscript
 0	string/b	#!\ /usr/local/bin/bash	Bourne-Again shell script text executable
 !:mime	text/x-shellscript
 
