Author: Daniel Novotny <dnovotny@redhat.com>
Description: one more PostScript font magic added.

diff -Naurp file.orig/magic/Magdir/fonts file/magic/Magdir/fonts
--- file.orig/magic/Magdir/fonts	2009-12-05 17:30:09.000000000 +0100
+++ file/magic/Magdir/fonts	2009-12-05 17:39:58.000000000 +0100
@@ -9,6 +9,9 @@
 0	string		%!PS-AdobeFont-1.	PostScript Type 1 font text
 >20	string		>\0			(%s)
 6	string		%!PS-AdobeFont-1.	PostScript Type 1 font program data
+0	string		%!FontType1		PostScript Type 1 font program data
+6	string		%!FontType1		PostScript Type 1 font program data
+0	string		%!PS-Adobe-3.0\ Resource-Font	PostScript Type 1 font text
 
 # X11 font files in SNF (Server Natural Format) format
 0	belong		00000004		X11 SNF font data, MSB first
@@ -58,7 +61,7 @@
 0	string		ttcf		TrueType font collection data
 
 # Opentype font data from Avi Bercovich
-0	string		OTTO		OpenType font data 
+0	string		OTTO		OpenType font data
 
 # Gürkan Sengün <gurkan@linuks.mine.nu>, www.linuks.mine.nu 
 0	string		SplineFontDB:	Spline Font Database 
